### PR TITLE
fix list-tagged-draggable for quotes !! and ##

### PR DESCRIPTION
**Pull request by BurningTreeC**

### CODE REVIEW

File: core/wiki/macros/list.tid [hunk 1]
```diff
@@ -50,28 +50,26 @@ tags: $:/tags/Macro
 </$vars>
 \end
 
-\define list-tagged-draggable-drop-actions()
+\define list-tagged-draggable-drop-actions(tag)
 <!-- Save the current ordering of the tiddlers with this tag -->
-<$set name="order" filter="[<tag>tagging[]]">
+<$set name="order" filter="[<__tag__>tagging[]]">
```
[the remark this answers is not in the document]
Comment (BurningTreeC) [reply core/wiki/macros/list.tid]: @Jermolene the `<__tag__>` formulation appears to be much better overall. I'm not sure if it's needed here but I wanted to go sure

[the remark this answers is not in the document]
Comment (BurningTreeC) [reply core/wiki/macros/list.tid]: @Jermolene - here the order of the tags in the dropdown gets stored before deleting any list-before and list-after fields in the next step. Then we enlist what's stored in `<<order>>` and use that to insert the actionTiddler before the currentTiddler

**Comment by BurningTreeC** [comment]
@Jermolene - I've quoted out a line that makes problems dragging `bad tags`

Doesn't the (new) line above (`<$action-listops $tiddler=<<__tag__>> $field="list" $filter="[enlist<order>] +[insertbefore:currentTiddler<actionTiddler>]"/>`) already assure that the tag is present?

**Comment by BurningTreeC** [comment]
Ah! now I get what this line is supposed to do. when dragging a tiddler title which is not tagged with that tag from outside into the tag-dropdown ...

I think I found a fix by replacing the mentioned line with:

```
<$list filter="[<actionTiddler>!contains:tags<__tag__>]">
<$fieldmangler tiddler=<<actionTiddler>>>
<$action-sendmessage $message="tm-add-tag" $param=<<__tag__>>/>
</$fieldmangler>
</$list>
```

**Comment by BurningTreeC** [comment]
I'm trying this with tags like `"""""" this !! is <<a>> // /very / $(baad)$ %!! $$taaaag$$ :: ??? {{something}} """`

**Comment by BurningTreeC** [comment]
... and it works, seems to be solid

**Comment by pmario** [comment]
I'm in favour, to make the title handling more robust but as written already, we should be sure about the consequences. see: #3517

**Comment by BurningTreeC** [comment]
@pmario yes I've seen your performance measurements... this fix here shouldn't affect the table-of-contents macros ... most of the changes are within the action when dropping something within a draggable list. As of now this is mostly used within the dropdowns of tag-pills iirc

**Comment by Jermolene** [comment]
Thanks @BurningTreeC

**Pull request merged**